pages/common/terminalizer.md

```diff
@@ -0,0 +1,23 @@
+# terminalizer
+
+> Utility program which records the terminal and generate animated gifs or share a video.
+
+- Create a global config directory:
```

Comment:
Does this create said directory in a pre-defined location, or in the current directory?

Reply:
Default location is `$HOME/.terminalizer`. I think it's known that all "global" configs goes into `$HOME`.

Reply:
Or you should think that need to be mentioned?

Reply:
Ah, I see. I don't think we need specify the directory it puts it in. What about tweaking it to this?

```suggestion
- Create the global config directory:
```

Using the definite article here helps imply that we're talking about 1 specific instance, instead 1 out of potentially many instances.